drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -197,6 +198,7 @@ def studio_view(self, context):
         fragment.initialize_js('DragAndDropEditBlock', {
             'data': self.data,
             'target_img_expanded_url': self.target_img_expanded_url,
+            'target_img_description': self.target_img_description,
```

Comment:
If this is already in `data.targetImgDescription `, is there any reason to pass it separately here?